polyfills.ts

```diff
@@ -0,0 +1 @@
+import 'fastestsmallesttextencoderdecoder';
```

Comment:
Pls add code comment explaining the purpose of this file

Reply:
Added comments, but ideally we would delete this file before sharing the repo. Should be handled by the Web3Modal dependency